index.js

```diff
@@ -25,17 +26,16 @@ exports.init = function (sbot, config) {
   }).map(function (file) {
     return ssbKeys.loadSync(path.join(dir, file))
   })
-  
+
   var keymap = {}
   var locks = {}
 
-  if(sbot.unboxers)
-    sbot.unboxers(function (content) {
-      for(var i = 0; i < keys.length; i++) {
-        var plaintext = _unbox(content, keys[i])
-        if(plaintext) return plaintext
-      }
-    })
+  sbot.addUnboxer(function(content) {
```

Comment:
I don't think you should follow automatically.
You might want to create a key without associating it with your main identity.
I think a better way to solve this problem is to have a way to check wether a new identity is followed by _someone_. (as shown in patchwork)

Reply:
expanding on that a bit, patchwork now shows you an alert if no one follows you.